packages/blade/src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { DropdownContext, useDropdown } from './useDropdown';
+import type { DropdownContextType } from './useDropdown';
+import { useId } from '~src/hooks/useId';
+import Box from '~components/Box';
+import { getPlatformType } from '~utils';
+
+type DropdownProps = {
+  selectionType?: 'single' | 'multiple';
+  children: React.ReactNode[];
+};
+
+const platformType = getPlatformType();
+const isReactNative = platformType === 'react-native';
+
+/**
+ * **Dropdown component**
+ *
+ * **Usage**
+ *
+ * ```jsx
+ * <Dropdown selectionType="single">
+ *  <SelectInput />
```

Comment:
Yes for Dropdown I think we can do.

I was thinking of avoiding it for ActionList components. There is lots of nesting there and already we have too much of runtime just to make the Dropdown work so trying to avoid in any JS possible.

For Dropdown I think it makes sense. Will do that since we just need 2 children. SelectInput and DropdownOverlay